torch/package/_directory_reader.py

```diff
@@ -25,7 +25,16 @@ def storage(self):
         return self._storage
 
 
-class MockZipReader(object):
+class DirectoryReader(object):
+    """
+    Class to allow PackageImporter to operate on unzipped packages. Methods
+    copy the behavior of the internal PyTorchFileReader class (which is used for
+    accessing packages in all other cases).
+
+    N.B.: ScriptObjects are not depickleable or accessible via this DirectoryReader
+    class due to ScriptObjects requiring an actual PyTorchFileReader instance.
```

Comment:
Nit: This docstring is missing proper markup for class names and references, but it won't be rendered in the API reference anyway so maybe it is better the way it is now.

Reply:
Yeah, I'll add the proper markups if we make it public. I didn't want to bother with doing the setup needed to test to make sure the rendering was correct if we're not actually going to render this